src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -282,6 +287,23 @@ def get_active_services(self) -> List[ServiceInfo]:
             for service in res.get("data")
         ]
 
+    def has_cluster_findings(self) -> bool:
+        res = (
+            self.client.table(ISSUES_TABLE)
+            .select('*')
+            .filter("account_id", "eq", self.account_id)
+            .filter("cluster", "eq", self.cluster)
+            .limit(1)
+            .execute()
+        )
+        if res.get("status_code") not in [200]:
+            msg = f"Failed to check cluster issues: {res.get('data')}"
+            logging.error(msg)
+            self.handle_supabase_error()
+            raise Exception(msg)
```

Comment:
This is a bit risky.
If there's some error here, we won't get the event history (which might be fine), but we won't start the cluster discovery at all. 
I think we don't want that

Reply:
nice catch